frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1533,15 +1537,18 @@ impl<T: Config> Pallet<T> {
 		<QueuedSolution<T>>::take()
 			.ok_or(ElectionError::<T>::NothingQueued)
 			.or_else(|_| {
-				T::Fallback::instant_elect(None, None)
-					.map_err(|fe| ElectionError::Fallback(fe))
-					.and_then(|supports| {
-						Ok(ReadySolution {
-							supports,
-							score: Default::default(),
-							compute: ElectionCompute::Fallback,
-						})
+				T::Fallback::instant_elect(
+					DataProviderBounds::new_unbounded(),
```

Comment:
Note that the fallback will have its own bounds configured by the runtime -- would be good to document this. 

Admittedly, this code looks strange, and we could improve this. 

What we want to express, is basically a `type ElectionBounds` and a `type FallbackBounds`.

Reply:
What you are suggesting here is to add a new config associated type to EPM like

```rust
type FallbackElectionBounds = Get<ElectionBounds>; 
```
which will be set to unbounded in this pallet (since we don't have it setup atm)?

And then here we can call

```rust
let fallback_bounds = T::FallbackElectionBounds::get();
T::Fallback::instant_elect(fallback_bounds.voters, fallback_bounds.targets);
```

I think this looks cleaner and will open the possibility for us to add the fallback on-chain election just by changing configs later on (as we discussed it's probably wise to get it eventually).